checkov/terraform/checks/resource/digitalocean/SpacesBucketVersioning.py

```diff
@@ -0,0 +1,19 @@
+
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class SpaceBucketVersioning(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure the Spaces bucket has versioning enabled"
+        id = "CKV_DIO_1"
+        supported_resources = ['digitalocean_spaces_bucket', 'digitalocean_spaces_bucket_object']
```

Comment:
```suggestion
        supported_resources = ['digitalocean_spaces_bucket']
```
I think you just meant to check the bucket for versioning 😉

Reply:
I did, id planned to do both which is what tfsec does but the object doesnt have a separate versioning attribute so their check is in error.